src/D2L.CodeStyle.Analyzers/Immutability/MutabilityAuditor.cs

```diff
@@ -3,15 +3,24 @@
 
 namespace D2L.CodeStyle.Analyzers.Immutability {
 	internal sealed class MutabilityAuditor {
-		public static bool IsAudited( ISymbol symbol, out Location location ) {
+		public static bool IsAudited( ISymbol symbol, out Location location, out Diagnostic diagnostic ) {
 			AttributeData attr;
+			diagnostic = null;
+
+			// Check if there is a conflicting audit AND unaudit
+			if( ( Attributes.Statics.Audited.IsDefined( symbol ) || Attributes.Mutability.Audited.IsDefined( symbol ) )
+				&& ( Attributes.Statics.Unaudited.IsDefined( symbol ) || Attributes.Mutability.Unaudited.IsDefined( symbol ) ) ) {

```

Comment:
It makes sense to me that the diagnostic goes on the identifier rather than an attribute, because we can't really choose which attribute to put it on.